### ISSUE TITLE
No documentation or testing for load_pe script

### ISSUE THREAD

**Issue by bocajnotnef** [ged-lab/khmer#784]
khmer/load_pe.py has no docs and no testing.

**Comment by ctb** [comment]
It can be removed; it's not used or publicized anywhere AFAICT.

**Comment by mr-c** [comment]
Confirmed

```
(env)mcrusoe@athyra:~/khmer/gl-master$ git grep load_pe
khmer/load_pe.py:def load_pe(screed_handle):
```

**Comment by ctb** [comment]
OK, so:

- [ ] remove khmer/load_pe.py

**Comment by kdmurray91** [comment]
Claiming this, see PR #828

**Comment by mr-c** [comment]
Fixed by #828
